Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -224,4 +229,15 @@ class EdgeHitProcessor: HitProcessing {
         return requestHeaders
     }
 
+    private func getCustomRequestPath(from event: Event) -> String? {
+        var customPath: String?
+        if let eventData = event.data {
+            let xdmData = eventData[EdgeConstants.JsonKeys.XDM] as? [String: Any]
```

Comment:
the new "request" key should be in the event data outside of the xdm payload as per spec

Reply:
Oh yes.